packages/@aws-cdk/aws-cloudfront-origins/README.md

```diff
@@ -76,3 +76,24 @@ new cloudfront.Distribution(this, 'myDist', {
 ```
 
 See the documentation of `@aws-cdk/aws-cloudfront` for more information.
+
+## Failover Origins (Origin Groups)
+
+You can set up CloudFront with origin failover for scenarios that require high availability.
+To get started, you create an origin group with two origins: a primary and a secondary.
+If the primary origin is unavailable, or returns specific HTTP response status codes that indicate a failure,
+CloudFront automatically switches to the secondary origin.
+You achieve that behavior in the CDK using the `OriginGroup` class:
+
+```ts
+new cloudfront.Distribution(this, 'myDist', {
+  defaultBehavior: {
+    origin: new origins.OriginGroup({
+      primaryOrigin: new origins.S3Origin(myBucket),
+      fallbackOrigin: new origins.HttpOrigin('www.example.com'),
+      // optional, defaults to: 500, 502, 503 and 504
+      fallbackStatusCodes: [501],
```

Comment:
I don't believe 501 is supported.

> You can choose any combination of the following status codes: 500, 502, 503, 504, 404, or 403.

(Source: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/high_availability_origin_failover.html)
```suggestion
      fallbackStatusCodes: [500],
```

Reply:
Oh wow, did not know that, thanks!